src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -96,68 +119,93 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                         notifications.ShowError(error?.Message ?? ex.Message);
                         return Observable.Empty<IPullRequestModel>();
                     }));
+            isExecuting = CreatePullRequest.IsExecuting.ToProperty(this, x => x.IsExecuting);
```

Comment:
Oh, and then looking 3 lines below I see it is. I missed that previously, it would've answered my question.